src/act-compat.js

```diff
@@ -2,29 +2,8 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import {reactDomSixteenPointNineIsReleased} from './react-dom-16.9.0-is-released'
 
-let reactAct
-let actSupported = false
-let asyncActSupported = false
-try {
```

Comment:
I'm not sure why this try catch block was here. was it to catch the error if reactAct was undefined? I'm guessing so.